### PR TITLE
Swahili Translations

### PR DESCRIPTION
**Pull request by Wangata**
## Type of Pull Request

- [x] Enhancement

Related Issue #s or links (if any): 

## Description of Changes
Added Swahili Translation

### CODE REVIEW

File: i18n/en/aspectRatio.json [hunk 1]
```diff
@@ -1,16 +1,16 @@
 {
-  "ratioPreview": "Ratio Preview",
```
Comment (dlford) [inline i18n/en/aspectRatio.json:2]: This one should be English, I think we got this file swapped with Swahili?

File: i18n/sw/aspectRatio.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (dlford) [inline i18n/sw/aspectRatio.json:1]: This one should be Swahili

**Review by dlford — CHANGES_REQUESTED** [review]
@Wangata great job here! One minor issue to fix. Also, please PR against the `dev` branch

File: i18n/en/aspectRatio.json (hunk 1 above)
Comment (Wangata) [reply i18n/en/aspectRatio.json:2]: Sure lemme work on it

**Pull request closed without merging**